webapp/components/TeaserImage/TeaserImage.vue

```diff
@@ -113,29 +114,41 @@ export default {
     },
     cropImage() {
       this.showCropper = false
+      if (this.file.type === 'image/png' || this.file.type === 'image/svg+xml') {
```

Comment:
Hm, what about other file extensions? Couldn't you check for `jpg` only here?

Reply:
I thought about that... I can change it